Sources/Tests/EndToEnd/Test_EndToEnd.c

```diff
@@ -72,7 +72,6 @@ static void ReadTemplate(const char *expectedFileName, Template *expectedTemplat
     assert(ret != EOF);
 }
 
-
```

Comment:
I got really confused about the `UnityFree` stuff - can someone explain why it is necessary?

I see that `malloc` and `free` are redefined, but what's it doing differently that means we have to duplicate code in our tests?